// koreanbots/dist/types.js
"use strict";
exports.__esModule = true;
var Category;
(function (Category) {
    Category[Category["\uAD00\uB9AC"] = 0] = "\uAD00\uB9AC";
    Category[Category["\uBBA4\uC9C1"] = 1] = "\uBBA4\uC9C1";
    Category[Category["\uC804\uC801"] = 2] = "\uC804\uC801";
    Category[Category["\uC6F9 \uB300\uC2DC\uBCF4\uB4DC"] = 3] = "\uC6F9 \uB300\uC2DC\uBCF4\uB4DC";
    Category[Category["\uB85C\uAE45"] = 4] = "\uB85C\uAE45";
    Category[Category["\uB3C4\uBC15"] = 5] = "\uB3C4\uBC15";
    Category[Category["\uAC8C\uC784"] = 6] = "\uAC8C\uC784";
    Category[Category["\uBC08"] = 7] = "\uBC08";
    Category[Category["\uB808\uBCA8\uB9C1"] = 8] = "\uB808\uBCA8\uB9C1";
    Category[Category["\uC720\uD2F8\uB9AC\uD2F0"] = 9] = "\uC720\uD2F8\uB9AC\uD2F0";
    Category[Category["\uBC88\uC5ED"] = 10] = "\uBC88\uC5ED";
    Category[Category["\uB300\uD654"] = 11] = "\uB300\uD654";
    Category[Category["NSFW"] = 12] = "NSFW";
    Category[Category["\uAC80\uC0C9"] = 13] = "\uAC80\uC0C9";
})(Category = exports.Category || (exports.Category = {}));
var Library;
(function (Library) {
    Library[Library["discord.js"] = 0] = "discord.js";
    Library[Library["Eris"] = 1] = "Eris";
    Library[Library["discord.py"] = 2] = "discord.py";
    Library[Library["discordcr"] = 3] = "discordcr";
    Library[Library["Nyxx"] = 4] = "Nyxx";
    Library[Library["Discord.Net"] = 5] = "Discord.Net";
    Library[Library["DSharpPlus"] = 6] = "DSharpPlus";
    Library[Library["Nostrum"] = 7] = "Nostrum";
    Library[Library["coxir"] = 8] = "coxir";
    Library[Library["DiscordGo"] = 9] = "DiscordGo";
    Library[Library["Discord4J"] = 10] = "Discord4J";
    Library[Library["Javacord"] = 11] = "Javacord";
    Library[Library["JDA"] = 12] = "JDA";
    Library[Library["Discordia"] = 13] = "Discordia";
    Library[Library["RestCord"] = 14] = "RestCord";
    Library[Library["Yasmin"] = 15] = "Yasmin";
    Library[Library["disco"] = 16] = "disco";
    Library[Library["discordrb"] = 17] = "discordrb";
    Library[Library["serenity"] = 18] = "serenity";
    Library[Library["SwiftDiscord"] = 19] = "SwiftDiscord";
    Library[Library["Sword"] = 20] = "Sword";
    Library[Library["\uAE30\uD0C0"] = 21] = "\uAE30\uD0C0";
    Library[Library["\uBE44\uACF5\uAC1C"] = 22] = "\uBE44\uACF5\uAC1C";
})(Library = exports.Library || (exports.Library = {}));
